scripts/update.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
```

Comment:
This scripts updates all modules to latest ones

Reply:
```
[submodule "packages/web-client-sdk"]
        path = packages/web-client-sdk
        url = https://github.com/fishjam-cloud/web-client-sdk.git
  branch = "main"

[submodule "packages/mobile-client-sdk"]
        path = packages/mobile-client-sdk
        url = https://github.com/fishjam-cloud/mobile-client-sdk.git
  branch = "main"
```
if we specify the branch in `.gitmodules` file, then running
```
git submodule update --remote --recursive
```
should do the job of this script

Reply:
I'm not sure about this solution: if we decide to add new method in SDK, we might want to release new version of documentation together with releasing new SDK version